diff --git a/python/sglang/srt/batch_invariant_ops/batch_invariant_ops.py b/python/sglang/srt/batch_invariant_ops/batch_invariant_ops.py
index 51a911e..58e4282 100644
--- a/python/sglang/srt/batch_invariant_ops/batch_invariant_ops.py
+++ b/python/sglang/srt/batch_invariant_ops/batch_invariant_ops.py
@@ -161,7 +161,7 @@ def matmul_kernel_persistent(
             accumulator += bias
         if c_ptr.dtype.element_ty == tl.float8e4nv:
             c = accumulator.to(tl.float8e4nv)
-        elif c_ptr.dtype.element_ty == tl.bfloat16:
+        if c_ptr.dtype.element_ty == tl.bfloat16:
             c = accumulator.to(tl.bfloat16)
         elif c_ptr.dtype.element_ty == tl.float32:
             c = accumulator.to(tl.float32)
@@ -713,7 +713,7 @@ def bmm_kernel_persistent(
 
         if c_ptr.dtype.element_ty == tl.float8e4nv:
             c = accumulator.to(tl.float8e4nv)
-        elif c_ptr.dtype.element_ty == tl.bfloat16:
+        if c_ptr.dtype.element_ty == tl.bfloat16:
             c = accumulator.to(tl.bfloat16)
         elif c_ptr.dtype.element_ty == tl.float32:
             c = accumulator.to(tl.float32)
